examples/wearable/UIComponents/contents/presentation-views/thumbnails/vertical-grid/vertical-circular-grid.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js">
+	</script>
+	<script src="../../../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page grid-page" data-enable-page-scroll="false">
```

Comment:
Generally, we use the id attribute for the page name in example.
So, I think id="grid-page" is better than the class="grid-page".